css/css-flexbox/percentage-padding-002-ref.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<link rel="author" title="Sergio Villar Senin" href="mailto:svillar@igalia.com">
+<style>
+div {
+    width: 50px;
+    height: 50px;
+}
+span {
+    display: block;
+    padding: 100%;
```

Comment:
IMO it would be better to avoid percentage paddings in the reference, could you replace it with pixel values?

Reply:
Sure. I'll have to specify padding on each element then because we have 2 `<span>` with 2 different paddings.